Uno.Themes.sln

```diff
@@ -7,8 +7,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Material", "src\library
 EndProject
 Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Uno.Themes.Samples.Shared", "src\samples\Uno.Themes.Samples\Uno.Themes.Samples.Shared\Uno.Themes.Samples.Shared.shproj", "{6279C845-92F8-4333-AB99-3D213163593C}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Uno.Themes.Samples.Droid", "src\samples\Uno.Themes.Samples\Uno.Themes.Samples.Droid\Uno.Themes.Samples.Droid.csproj", "{46670BA5-CDEC-4187-8D99-74ADDE56E4DC}"
-EndProject
```

Comment:
Was this supposed to be removed?

Reply:
no, visual studio unwanted changes, removed. Thanks!